data-model/aliquot.yaml

```diff
@@ -0,0 +1,88 @@
+id: https://23id.org/anvil/aliquot 
+name: anvil-aliquot-schema
+prefixes:
+  anvil: 
+    prefix_prefix: anvil
+    prefix_reference: https://w3id.org/anvil/
+  linkml:
+    prefix_prefix: linkml
+    prefix_reference: https://w3id.org/linkml/
+  ig2-biospecimen-availability: 
+    prefix_prefix: ig2-biospecimen-availability 
+    prefix_reference: https://nih-ncpi.github.io/ncpi-fhir-ig-2/CodeSystem/biospecimen-availability
+imports:
+  - linkml:types
+  - anvil_core
+classes: 
+  Aliquot:
+    title: Aliquot
+    definition_uri: anvil:aliquot 
+    annotations:  
+      required:  
```

Comment:
Is the `required` annotation one of the non-LinkML things that was from Schematic? I'm not sure I understand what purpose it is serving.